### ISSUE TITLE
GENERAL: species places

### ISSUE THREAD

**Issue by solpahi** [CvGC/dict#20]
Every animal/plant gismu has an x2 that specifies its species or breed. The one exception is remna, which has no x2, even though it is an animal.

Still, one could say the species places are regular, and thus not harmful.

However, to me it feels a bit as though insisting that a living thing must have a species classification x2 is akin to saying that the most important thing about, say, a dog, is which species or breed it belongs to. Like animals are collectors' items whose purpose is to be classified, but who don't matter beyond that. 

Humans have a strong drive to classify things... I mean animals. Non-animal gismu do not have an "of category x2" place, while remna only refers to the species Homo sapiens (sapiens), because classifying humans would be evil, but non-humans are fair game.

**ko'a gerku gi'e jutlei ko'e**
_"X is a dog that belongs to the species Y."_

**Comment by este-rs** [comment]
I agree with removing the x2, though I'd add to the notes that x2 can be
used for species.

**Comment by mezohe** [comment]
No opinion on whether the species places should be removed in general, but {remna} should be consistent with the others.
